packages/duppy/duppy.0.7.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "Romain Beauxis <toots@rastageeks.org>"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+homepage: "https://github.com/savonet/ocaml-duppy"
+build: [
+  ["./configure" "--prefix" prefix]
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: ["ocamlfind" "remove" "duppy"]
+depends: [
+  "camlp4"
+  "ocamlfind"
+  "pcre"
+]
+depopts: [
+  "ssl"
+  "osx-secure-transport"
+]
+conflicts: ["liquidsoap" {<= "1.2.1"}]
+bug-reports: "https://github.com/savonet/ocaml-duppy/issues"
```

Comment:
I think this needs a `ocaml-version >= "4.02..3"` constraint in the `available:` field

```
# File "duppy.ml", line 479, characters 8-27:
# Error: Unbound value Buffer.add_subbytes
# make[2]: *** [duppy.cmo] Error 2
# make[1]: *** [byte-code-library] Error 2
# make: *** [all] Error 2
```

Reply:
That's right, fixing it, thanks!